src/status_im2/common/bottom_sheet_screen/view.cljs

```diff
@@ -71,12 +70,18 @@
            [reanimated/view {:style (style/background opacity)}])
          [gesture/gesture-detector
           {:gesture (drag-gesture translate-y opacity scroll-enabled curr-scroll close)}
-          [reanimated/view {:style (style/main-view translate-y override-theme)}
+          [reanimated/view {:style (style/main-view translate-y theme)}
            [rn/view {:style style/handle-container}
-            [rn/view {:style (style/handle override-theme)}]]
+            [rn/view {:style (style/handle theme)}]]
            [content
             {:insets         insets
              :close          close
              :scroll-enabled scroll-enabled
              :current-scroll curr-scroll
              :on-scroll      #(on-scroll % curr-scroll)}]]]]))))
+
+(defn- view
+  [params]
+  [:f> f-view params])
+
+(def themed-view (theme/with-theme view))
```

Comment:
To me, the public var name should be as stable as possible. If it's themed or not the name should be just `view`. In the future, we may have other HOFs and other props might be "injected", but the var `view` should still be the same.

In another PR I saw the dev used `internal-view` (private) for the non-wrapped component and `view` for the wrapped one with `theme/with-theme`. I prefer this way because it's a good practice to keep APIs more stable.

Reply:
Agreed and updated 👍